### PR TITLE
fix: typo spelling grammar

### PR DESCRIPTION
**Pull request by slowy07**
**Description**
[minor fix] typo spelling grammar, and replace to correct with reference from [Merriam webster](merriam-webster.com) and [wikitionary](https://www.wiktionary.org/)
**testing**
no testing because just replace the miss grammar word with the corret one

**Checklist**
- [x] added to README
- [x] route with openpilot:
- [x] route with stock system:

### CODE REVIEW

**Comment by pd0wm** [comment]
No need to fix typos in third party code. Anything in `phonelibs` and 'pyextra` can be ignored.

**Comment by slowy07** [comment]
yes sir @pd0wm, no need to fix the typos in specified files ```phonelibs``` and ```pyextra``` right ??

File: selfdrive/locationd/calibrationd.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def update_status(self):
       self.cal_status = Calibration.INVALID
 
     # If spread is too high, assume mounting was changed and reset to last block.
```
Comment (grekiki) [inline selfdrive/locationd/calibrationd.py:117]: This is still erong

File: selfdrive/locationd/calibrationd.py [hunk 2]
```diff
@@ -115,7 +115,7 @@ def update_status(self):
...
-    # Make the transition smooth. Abrupt transistion are not good foor feedback loop through supercombo model.
+    # Make the transition smooth. Abrupt transition are not good foor feedback loop through supercombo model.
```
Comment (grekiki) [inline selfdrive/locationd/calibrationd.py:118]: This is still wrong

Comment (adeebshihadeh) [reply selfdrive/locationd/calibrationd.py:118]:
```suggestion
    # Make the transition smooth. Abrupt transitions are not good foor feedback loop through supercombo model.
```

**Pull request merged**